src/main/java/net/countercraft/movecraft/combat/features/tracking/KillCredCommand.java

```diff
@@ -0,0 +1,39 @@
+package net.countercraft.movecraft.combat.features.tracking;
+
+import net.countercraft.movecraft.combat.localisation.I18nSupport;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+
+public class KillCredCommand implements CommandExecutor {
+    @Override
+    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, @NotNull String... args) {
+        if (sender instanceof Player) {
+            Player p = (Player) sender;
+            if (p.hasPermission("movecraft.combat.hidekillcred")) {
+                if (DamageTracking.getKillCredMap().get(p) == null) {
+                    DamageTracking.setKillCredMap(p,true);
+                    p.sendMessage(I18nSupport.getInternationalisedString("You are now hiding your kills."));
+                    return true;
+                }
+
+                if (DamageTracking.getKillCredMap().get(p) == true) {
```

Comment:
With the above changes, there will be no need for the `== true` here.  You'll be able to put a single if statement of the following form:
```java
if(DamageTracking.getKillCred(p)) {
  // actions if hidden
}
else {
  // actions if not hidden
}
```

We can also break the `return true` statements out of the if statement, since it is in both branches.